pom.xml

```diff
@@ -1,102 +1,102 @@
+            <artifactId>spring-boot-starter-test</artifactId>
+            <scope>test</scope>
+        </dependency>
 
-    <dependency>
-      <groupId>org.spockframework</groupId>
-      <artifactId>spock-core</artifactId>
-      <version>${spock.version}</version>
-      <scope>test</scope>
-    </dependency>
+        <dependency>
+            <groupId>org.spockframework</groupId>
+            <artifactId>spock-core</artifactId>
+            <version>${spock.version}</version>
+            <scope>test</scope>
+        </dependency>
 
-    <dependency>
-      <groupId>org.spockframework</groupId>
-      <artifactId>spock-spring</artifactId>
-      <version>${spock.version}</version>
-      <scope>test</scope>
-    </dependency>
+        <dependency>
+            <groupId>org.spockframework</groupId>
+            <artifactId>spock-spring</artifactId>
+            <version>${spock.version}</version>
+            <scope>test</scope>
+        </dependency>
 
-  </dependencies>
+    </dependencies>
 
-  <build>
-  <plugins>
-    <plugin>
-      <groupId>org.codehaus.gmavenplus</groupId>
-      <artifactId>gmavenplus-plugin</artifactId>
-      <version>2.0.0</version>
-      <executions>
-        <execution>
-          <goals>
-            <goal>compile</goal>
-            <goal>compileTests</goal>
-          </goals>
-        </execution>
-      </executions>
-    </plugin>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.gmavenplus</groupId>
+                <artifactId>gmavenplus-plugin</artifactId>
+                <version>2.0.0</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>compile</goal>
+                            <goal>compileTests</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
 
-    <plugin>
-      <groupId>org.apache.maven.plugins</groupId>
-      <artifactId>maven-surefire-plugin</artifactId>
-      <version>3.0.0-M7</version>
-      <configuration>
-        <useModulePath>false</useModulePath>
-        <useFile>false</useFile>
-        <includes>
-          <include>**/*Test</include>
-          <include>**/*Spec</include>
-        </includes>
-      </configuration>
-    </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>3.0.0-M7</version>
+                <configuration>
+                    <useModulePath>false</useModulePath>
+                    <useFile>false</useFile>
+                    <includes>
+                        <include>**/*Test</include>
+                        <include>**/*Spec</include>
+                    </includes>
+                </configuration>
+            </plugin>
 
-  </plugins>
-  </build>
+        </plugins>
+    </build>
 </project>
```

Comment:
Personally I prefer two spaces indents, but if you want we can stick to four

Reply:
It's default intellij formatting, I think we should have same code style to avoid common problems